Create add_music.md

KarenWest opened the pull request:
"Please review this, @casey-collab"

I am participating in the non-pay version of the class, and it asked that we make a change to the "create your own adventure" repository, so I added the first stanza lyrics from Queen's "Killer Queen", since it's the only song lyrics I could think of related to cake eating and champagne. ;-)  I know the directions said that the non-pay participants may not see the changes made, but I thought I'd add something just in case.  No one added music so I thought I would (related musical lyrics).

<!-- Reviewable:start -->
[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/udacity/create-your-own-adventure/1637)
<!-- Reviewable:end -->

casey-collab commented on english/add_music.md:
```diff
@@ -0,0 +1,11 @@
+Add Music for your cake recipe?
```
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

casey-collab replied:
It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

KarenWest replied:
This may be a stupid question - but how do you first create a directory on github?  I don't recall Caroline and Sarah saying how they did that - perhaps I missed it - thanks. ;-)

KarenWest replied:
I just did a search - you first have to clone to your local machine in order to create a directory, since github only allows you to create files on the web site?  I'll try that.

cbuckey-uda replied:
@KarenWest, I had never created a directory on the GitHub website before. I always clone and make all changes locally. However, I found [this post](http://stackoverflow.com/questions/18773598/creating-folders-inside-github-com-repo-without-using-git) with instructions on how to create a directory on GitHub. It's not a very intuitive way, but I just tried it in a test repository and it worked for me. Of course, cloning and creating the directory on your own computer is a great way to get more practice with remotes and pushing.

casey-collab commented:
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

KarenWest commented:
"Please review this, @casey-collab"

Below is my earlier comment.  I've since created a music directory under the english directory for create-your-own-adventure, and placed the add_music.md file there, and deleted it from the english directory.

I am participating in the non-pay version of the class, and it asked that we make a change to the "create your own adventure" repository, so I added the first stanza lyrics from Queen's "Killer Queen", since it's the only song lyrics I could think of related to cake eating and champagne. ;-) I know the directions said that the non-pay participants may not see the changes made, but I thought I'd add something just in case. No one added music so I thought I would (related musical lyrics).

casey-collab commented on english/music/add_music.md:
```diff
@@ -0,0 +1,11 @@
+Add Music for your cake recipe?
```
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

KarenWest replied:
I do not know how to do this - I'll just skip it for now - perhaps edit an existing file and add these lines, since I'm a web novice - still learning.  Thanks.

KarenWest replied:
I figured it out - let me know if this addition is acceptable!  I added a link to the:
create-your-own-adventure/dance/bake-a-cake/bake-a-cake.md file that has a link to:
create-your-own-adventure/music/add_music.md file, that contains the Queen song "Killer Queen" lyrics, about letting them eat cake, just like Mary Antoinette, and champagne, etc.  Hope it's correctly done this time! ;-)

cbuckey-uda replied:
Yes, it looks like you figured it out. I'm glad to see it!

casey-collab commented:
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

cbuckey-uda commented:
@casey-collab, please take another look

casey-collab commented:
Looks great! :smiley_cat:
